lib/elixir_google_scraper/scraper.ex

```diff
@@ -0,0 +1,38 @@
+defmodule ElixirGoogleScraper.Scraper do
+  @moduledoc """
+  The Scraper context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias ElixirGoogleScraper.Accounts.User
+  alias ElixirGoogleScraper.Repo
+  alias ElixirGoogleScraper.Scraper.CSVKeyword
+  alias ElixirGoogleScraper.Scraper.Keyword
+
+  def create_keyword(attrs \\ %{}) do
+    %Keyword{}
+    |> Keyword.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def save_keywords(file, %User{} = user) do
+    case CSVKeyword.validate(file) do
+      {:ok, keyword_list} ->
+        Enum.each(keyword_list, fn keyword ->
+          create_keyword(%{
+            name: List.first(keyword),
+            user_id: user.id
+          })
+        end)
+
+        {:ok, :file_is_proccessed}
```

Comment:
There is only one possible `:ok` case so I think we can omit `:file_is_proccessed` and return only `:ok` without tuple